mlflow/models/docker_utils.py

```diff
@@ -71,7 +71,11 @@ def _get_mlflow_install_step(dockerfile_context_dir, mlflow_home):
             "RUN mvn "
             " --batch-mode dependency:copy"
             " -Dartifact=org.mlflow:mlflow-scoring:{version}:jar"
-            " -DoutputDirectory=/opt/java/jars"
+            " -DoutputDirectory=/opt/java/jars\n"
+            "RUN cp /opt/java/mlflow-scoring-{version}.pom /opt/java/pom.xml\n"
```

Comment:
Set up a `/opt/java/pom.xml` file so that we can invoke `mvn copy-dependencies` ([see docs](http://maven.apache.org/plugins/maven-dependency-plugin/copy-dependencies-mojo.html)) from the `/opt/java` directory (treat `/opt/java` as a mvn project)